<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>559</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="baseSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>DirWizard</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="2">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_Add_Dir">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>45</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="toolTip">
             <string>Add folder</string>
            </property>
            <property name="statusTip">
             <string>Add folder to path list.</string>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_Remove_Dir">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>45</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="toolTip">
             <string>Remove selected folder</string>
            </property>
            <property name="statusTip">
             <string>Remove folder from path list.</string>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <widget class="QListWidget" name="listWidget">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>90</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16770</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>Folders for processing</string>
          </property>
          <property name="statusTip">
           <string>Path list: every path in list will be processed. You can Drag&amp;Drop folder to here.</string>
          </property>
          <property name="resizeMode">
           <enum>QListView::Fixed</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_Folders">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Folders:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="formLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <property name="leftMargin">
         <number>8</number>
        </property>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_Duplicate_Search">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>Start duplicate search</string>
          </property>
          <property name="statusTip">
           <string>Start duplicate files search</string>
          </property>
          <property name="text">
           <string>Start Dupl Find</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_Cancel">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>Cancel current process</string>
          </property>
          <property name="statusTip">
           <string>Cancel current process.</string>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="useFiltersBtn">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>Push to use file filters.</string>
          </property>
          <property name="statusTip">
           <string>Push to use file filters.</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton:checked{ background-color:rgb(1, 39, 255); }
</string>
          </property>
          <property name="text">
           <string>Use filters</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="setFiltersBtn">
          <property name="toolTip">
           <string>Push to change file filters</string>
          </property>
          <property name="statusTip">
           <string>Push to change file filters</string>
          </property>
          <property name="text">
           <string>Set filters</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="label_Result">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Result:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QTableView" name="tableView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>780</width>
          <height>256</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Result table</string>
        </property>
        <property name="statusTip">
         <string>Table with results.</string>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton_Remove_Checked">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>Remove elements, where are you make check from disk</string>
          </property>
          <property name="statusTip">
           <string>Remove checked elements from disk.</string>
          </property>
          <property name="text">
           <string>Remove checked</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Save_From_Table">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Save items from table to file.</string>
          </property>
          <property name="statusTip">
           <string>Save items from table to file.</string>
          </property>
          <property name="text">
           <string>Save to File</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Progress.</string>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="textVisible">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSaveAs"/>
   </widget>
   <widget class="QMenu" name="menuWork_with_results">
    <property name="title">
     <string>Work with results</string>
    </property>
    <addaction name="actionClear_all_flags"/>
    <addaction name="actionRemove_checked"/>
    <addaction name="actionSelect_files_in_concrete_folder"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuWork_with_results"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="iconText">
    <string>About</string>
   </property>
   <property name="statusTip">
    <string>About this program.</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="actionStart_Dupl_Find">
   <property name="text">
    <string>Start Dupl Find</string>
   </property>
   <property name="toolTip">
    <string>Start duplicate search</string>
   </property>
   <property name="statusTip">
    <string>Start duplicate files search</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="actionCompare_Folders">
   <property name="text">
    <string>Compare Folders</string>
   </property>
   <property name="toolTip">
    <string>Compare folders, find uniq files.</string>
   </property>
   <property name="statusTip">
    <string>Compare folders and find uniq files.</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="actionCheck_ZIP">
   <property name="text">
    <string>Check ZIP</string>
   </property>
   <property name="toolTip">
    <string>Check ZIP archives</string>
   </property>
   <property name="statusTip">
    <string>Check ZIP archives. Show invalid or bad ZIP archives.</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="actionCheck_Hashes">
   <property name="text">
    <string>Check Hashes</string>
   </property>
   <property name="toolTip">
    <string>Check hashes</string>
   </property>
   <property name="statusTip">
    <string>Check files in folders by hashes.</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="actionCalc_Hashes">
   <property name="text">
    <string>Calc Hashes</string>
   </property>
   <property name="toolTip">
    <string>Calc hashes and store to disk.</string>
   </property>
   <property name="statusTip">
    <string>Calc hash sum for each file in folders and  and save to disk.</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open saved duplicates file list.</string>
   </property>
   <property name="statusTip">
    <string>Open saved duplicates file list.</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save results from table to file.</string>
   </property>
   <property name="toolTip">
    <string>Save results as duplicates files list from table to file.</string>
   </property>
   <property name="statusTip">
    <string>Save results from table to file.</string>
   </property>
  </action>
  <action name="actionClear_all_flags">
   <property name="text">
    <string>Clear all selection flags.</string>
   </property>
   <property name="statusTip">
    <string>Clear all selection flags.</string>
   </property>
  </action>
  <action name="actionRemove_checked">
   <property name="text">
    <string>Remove checked</string>
   </property>
   <property name="iconText">
    <string>Remove checked elements from disk.</string>
   </property>
   <property name="toolTip">
    <string>Remove elements, where are you make check from disk</string>
   </property>
   <property name="statusTip">
    <string>Remove checked elements from disk</string>
   </property>
  </action>
  <action name="actionSelect_files_in_concrete_folder">
   <property name="text">
    <string>Mark files in concrete folder as checked.</string>
   </property>
   <property name="statusTip">
    <string>Mark files in concrete folder as checked.</string>
   </property>
  </action>
  <action name="actionFind_Empty_folders">
   <property name="text">
    <string>Find Empty folders</string>
   </property>
   <property name="statusTip">
    <string>Find Empty folders</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
